sys/shell/src/shell.c

```diff
@@ -470,6 +470,11 @@ get_command_from_module(const char *command, int len, int module)
 
     shell_module = &shell_modules[module];
     for (i = 0; shell_module->commands[i].sc_cmd; i++) {
+
+        if (strlen(shell_module->commands[i].sc_cmd) != len) {
```

Comment:
I think using `strcmp` instead of `strncmp` below would basically do the same :)

Reply:
discussed offline: `command` is not null-terminated thus need for `strncmp` below